### ISSUE TITLE
Unexpected keyword argument passed to optimizer: schedule_decay

### ISSUE THREAD

**Issue by eduardobellotto** [SciSharp/Keras.NET#111]
Hi.

When I try to create a nadam optimizer with a specific learning rate:
```
var optimizer = new Keras.Optimizers.Nadam(lr: 0.0001f);
```
I get the following error:

Python.Runtime.PythonException: 'TypeError : Unexpected keyword argument passed to optimizer: schedule_decay'

Am I doing something wrong?

Thanks.

**Comment by deepakkumar1984** [comment]
Fixed in latest code, Will update nuget today

**Comment by eduardobellotto** [comment]
Thank you very much!
